internal/log/log.go

```diff
@@ -41,6 +48,7 @@ func SetLogLevel(logLevel string, debug bool) {
 
 	if debug {
 		logrus.SetLevel(logrus.DebugLevel)
+		SetDebugFormatter()
```

Comment:
Debug is not on by default, so I am wondering why you put that under debug ?

Reply:
I have changed the default formatter code to display only the error detail. However, for debugging it would be helpful to have the line number and some more details about the error. Hence, I created a new debug formatter and setting it only when the log level is `debug`.

Reply:
That the easy formatter right ?  but it seems you set it only when debug is enabled .. sorry I may missed something

Reply:
The easy formatter is called here https://github.com/yugabyte/ybm-cli/blob/550650581ce3ee84742de6852692fbae8d661e63/cmd/root.go#L131, the debug formatter is called only when the log level is `debug`.

Reply:
Oh lol I am still sleepy it seems :)
Thanks